scripts/zones/RoMaeve/npcs/Goblin_Footprint.lua

```diff
@@ -0,0 +1,66 @@
+-----------------------------------
+-- Area: Bostaunieux Oubliette
+--  NPC: Goblin Footprint
+-- Type: NPC
+-- !pos  111.806 -24.007 57.266 167
+-----------------------------------
+local ID = require("scripts/zones/RoMaeve/IDs")
+require("scripts/globals/pasteventwatcher")
+require("scripts/globals/quests")
+require("scripts/globals/missions")
+-----------------------------------
+---
```

Comment:
Don't need the ---

Reply:
removed

Reply:
looks like one of your followup commits may have put it back? there is also another above the other onTrade

Reply:
@TeoTwawki  thanks for catch, yeah i PR'd wrong branch lol xD
It's fixed now =)